vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors_moe.py

```diff
@@ -106,41 +106,13 @@ class CompressedTensorsMoEMethod(FusedMoEMethodBase):
     def get_moe_method(
         quant_config: "CompressedTensorsConfig",  # type: ignore # noqa E501
```

Comment:
we can get rid of quant_config entirely here since we have quant_scheme but it would require us converting 5 or so CompressedTensorsConfig.is_X methods to class/static methods at which point i'd probably just change them all. Want to make sure i'm on the right track before i start doing all that.